bitvm/src/signatures/wots_api.rs

```diff
@@ -144,7 +144,9 @@ macro_rules! impl_wots {
         }
     };
 }
+const BIGINT_LEN: u32 = 32;
+pub const HASH_LEN: u32 = 20; // bytes, can lower it to value like 16 or 13 lesser acceptable security
 
 // Expand the macro for the two variants.
-impl_wots!(wots160, 20);
-impl_wots!(wots256, 32);
\ No newline at end of file
+impl_wots!(wots_hash, HASH_LEN);
```

Comment:
this is the only significant change in this PR.
The rest are naming changes from _160 to _hash.
To change to a lower hash length, only change HASH_LEN param above.
example: set HASH_LEN: u32 = 16 for 16 byte hash.